lib/utils/pyexec.c

```diff
@@ -450,6 +487,8 @@ int pyexec_friendly_repl(void) {
     */
 
     for (;;) {
+        mp_hal_stdio_mode_raw();
```

Comment:
what does this change?

Reply:
it's needed only by the unix port to switch to/from raw console mode